Doc/whatsnew/3.10.rst

```diff
@@ -1746,7 +1746,7 @@ Deprecated
 
   * ``threading.Thread.setDaemon`` => :attr:`threading.Thread.daemon`
 
-  (Contributed by Jelle Zijlstra in :issue:`21574`.)
+  (Contributed by Jelle Zijlstra in :issue:`43723`.)
```

Comment:
Why not just linking the corresponding GH issue?
```suggestion
  (Contributed by Jelle Zijlstra in :gh:`87889`.)
```

Reply:
Good idea!